Doc/tools/extensions/pyspecific.py

```diff
@@ -131,6 +131,26 @@ def run(self):
         return [pnode]
 
 
+# Support for documenting platform availability
+
+class Availability(Directive):
+
+    has_content = False
+    required_arguments = 1
+    optional_arguments = 0
+    final_argument_whitespace = True
+
+    def run(self):
+        pnode = nodes.paragraph(classes=['availability'])
+        n, m = self.state.inline_text(':ref:`Availability <availability>`: ',
+                                      self.lineno)
+        pnode.extend(n + m)
+        n, m = self.state.inline_text(self.arguments[0], self.lineno)
+        pnode.extend(n + m)
+        pnode += nodes.Text('.', '.')
```

Comment:
I dislike the idea of adding a trailing dot in the macro. Can you please remove it? I prefer to give most freedom to the user of the macro.

Reply:
Removed.